source/includes/action.md

```diff
@@ -56,7 +54,7 @@ State | Description
 ----- | -----------
 In progress | The action needed asynchronous execution and is being performed
 Success | The action was executed successfully
-Failed | There was an issue when the action was being performed. Check the logs for more information.
+Failed | There was an issue when the action was being performed. Check the logs for more information (missing pointer for tutum-live doc).
```

Comment:
done in https://github.com/tutumcloud/api-docs/commit/70089c1a283812cff79fb0c849c7a17aca857a0d